MSAL/src/native_auth/controllers/MSALNativeAuthBaseController.swift

```diff
@@ -163,4 +163,11 @@ class MSALNativeAuthBaseController {
             }
         }
     }
+
+    func updateContext(_ previousContext: MSIDRequestContext, with correlationId: UUID?) -> MSALNativeAuthRequestContext {
+        return MSALNativeAuthRequestContext(
```

Comment:
If the correlationId is nil why not just return the `previousContext` instead of instantiating a new one?

Reply:
The return type is `MSALNativeAuthRequestContext` because the SignIn and Token flows were using it instead of `MSIDRequestContext`. 

But you're right: it makes more sense to return the same instance if there aren't any changes needed, so I've updated these flows to accept an `MSIDRequestContext` instead.